test/default_integration_plan_new_test.go

```diff
@@ -0,0 +1,104 @@
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"test/validation"
+)
+
+type TestParams struct {
```

Comment:
I think this will be cleaner if you move the struct definition and the test definitions into TestPlanDefaults. Among other reasons, the test definition move will enable per-test invocations in IDEs that look for the common golang subtest pattern.

Reply:
I suspect you can make these struct members private, too.